File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        unordered_map<char, int> mp;
        for(int i = 0; i < s.length(); i++){
            mp[s[i]]++;
        }
        for(int i = 0; i < t.length(); i++){
            if(mp[t[i]]) mp[t[i]]--;
            else return false;
        }
        for(pair<char, int> elem : mp){
            if(elem.second > 0) return false;
            //cout << elem.second << endl;
        }
        return true;
    }    
};

File: 0242-valid-anagram/0242-valid-anagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        //  https://leetcode.com/problems/valid-anagram/solutions/4410240/three-lines-of-code-by-prodonik-java-c-python-go-js-ruby-c
        //  나는 hashtable을 이용 했지만
        //  이 풀이는 문자열을 정렬해 비교
        char[] ss = s.ToCharArray();
        char[] tt = t.ToCharArray();

        Array.Sort(ss);
        Array.Sort(tt);

        return new string(ss) == new string(tt);
    }
}